tests/comm/test_trtllm_mnnvl_allreduce.py

```
@@ -17,10 +19,56 @@
     init_torch_distributed_from_mpi,
 )
+from tests.test_helpers.utils_fp4 import (
+    cast_from_fp4,
+    recover_swizzled_scales,
+    ref_fp4_quant,
+)
 
 # Note: torch.distributed cleanup is handled by tests/comm/conftest.py
 
 
+def fp8_quant(input: torch.Tensor, scale: torch.Tensor) -> torch.Tensor:
```

Ruff (0.15.13)

[error] Function argument `input` is shadowing a Python builtin

(A002)